manifests/init.pp

```diff
@@ -93,8 +93,9 @@
     validate_absolute_path($ssl_cert_db, $ssl_cert_password_file)
   }
 
-  class { '::qpid::install': } ~>
-  class { '::qpid::config': } ~>
-  class { '::qpid::service': } ->
-  Class['qpid']
+  include ::qpid::install
```

Comment:
It does mean that if you want something to happen before the install, you're not sure it is contained.